cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -40,46 +40,51 @@ In `{odo-title}`, services are provisioned from the OpenShift Service Catalog an
 [id="odo-supported-languages-and-images"]
 === Officially supported languages and corresponding container images
 
-.Supported languages, container images, and package managers
+.Supported languages, container images, package managers, and platforms
 [options="header"]
 |===
-|Language |Container image |Package manager
+|Language |Container image |Package manager |Platform
 |*Node.js*
-|https://registry.centos.org/centos/nodejs-10-centos7/latest[centos/nodejs-10-centos7]
-|NPM
-
-|
-|https://hub.docker.com/r/centos/nodejs-12-centos7[centos/nodejs-12-centos7]
+|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-10-rhel7[rhscl/nodejs-10-rhel7]
 |NPM
+|amd64, s390x
 
 |
-|https://catalog.redhat.com/software/containers/rhscl/nodejs-12-rhel7/[rhscl/nodejs-12-rhel7]
+|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-12-rhel7[rhscl/nodejs-12-rhel7]
 |NPM
+|amd64, s390x
 
 |
-|https://www.github.com/bucharest-gold/centos7-s2i-nodejs[bucharestgold/centos7-s2i-nodejs]
+|https://access.redhat.com/articles/3376841[rhoar-nodejs/nodejs-10]
 |NPM
+|amd64
 
 |
-|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-8-rhel7[rhscl/nodejs-8-rhel7]
+|https://github.com/sclorg/s2i-nodejs-container[centos/nodejs-10-centos7]
```

Comment:
@SNiemann15 @boczkowska Is there a reason that these are going from a red hat registry image to now pointing to github links? That doesn't seem right to me - aren't there downstream images for these?

Reply:
These are the images that odo QE provided me with, cc @amitkrout

Reply:
It is downstream-ready from what I understand
`NodeJS images based on Red Hat Software Collections and intended for OpenShift and general usage, that provide a platform for building and running NodeJS applications. Users can choose between Red Hat Enterprise Linux, Fedora, and CentOS based images. `

Reply:
@boczkowska the correct RH downstream link to that should be: https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-10-rhel7.

The GH repo link is technically not supported and we should replace it with the Red Hat registry image one. I am not sure why QE says that is the right image for downstream (the link is to CentOS image, which is **_definitely_** not correct and not supported by RH).

@sferich888 if you have thoughts on this, please feel free to add.